build.csx

```diff
@@ -153,7 +153,7 @@ targets.Add(
 
 targets.Add(
     "pack",
-    DependsOn("build", "outputDirectory"),
+    DependsOn("build", "pdbgit"),

```

Comment:
I liked the removal of `outputDirectory` until I realized that `build` doesn't actually use it. Maybe keep here and remove from the `build` dependencies?